pr2_bringup/CMakeLists.txt

```diff
@@ -24,3 +21,16 @@ install(FILES
    pr2_recalibrate.launch
    pr2-c2.launch
    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+
+if (CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
+  find_package(roslaunch REQUIRED)
+  # Tests that PR2 diagnostic analyzers configuration loads
+  add_rostest(test/pr2_analyzers_load_test.launch)
+  set(roslaunch_check_script ${PROJECT_SOURCE_DIR}/test/roslaunch-check)
```

Comment:
Why did you copy&override the script from `roslaunch` for this?
I guess there was a good reason? In that case please add it as a comment somewhere.